apps/web/src/components/HyloEditor/HyloEditor.js

```diff
@@ -34,100 +34,124 @@ const HyloEditor = React.forwardRef(({
-
-                  try {
-                    const url = new URL(href)
-
-                    element.innerHTML = `${url.hostname}${url.pathname !== '/' ? url.pathname : ''}`
-                    return element
-                  } catch (e) {
-                    element.innerHTML = href
-                    return element
-                  }
+    StarterKit.configure({
+      heading: {
+        levels: [1, 2, 3]
+      }
+    }),
+
+    Placeholder.configure({ placeholder }),
+
+    Link.extend({
+      // This expands concatenated links back to full href for editing
+      parseHTML () {
+        return [
+          {
+            tag: 'a',
+            // Special handling for links who's innerHTML has been concatenated by the backend
+            contentElement: element => {
+              if (element.innerHTML.match(/…$/)) {
+                const href = element.getAttribute('href')
+
+                try {
+                  const url = new URL(href)
+
+                  element.innerHTML = `${url.hostname}${url.pathname !== '/' ? url.pathname : ''}`
+                  return element
+                } catch (e) {
+                  element.innerHTML = href
+                  return element
                 }
-
-                return element
               }
+
+              return element
             }
-          ]
-        },
-        addOptions () {
-          return {
-            ...this.parent?.(),
-            openOnClick: false,
-            autolink: true,
-            HTMLAttributes: {
-              target: null
-            },
-            validate: href => {
-              onAddLink && onAddLink(href)
-              return true
-            }
+          }
+        ]
+      },
+      addOptions () {
+        return {
+          ...this.parent?.(),
+          openOnClick: false,
+          autolink: true,
+          HTMLAttributes: {
+            target: null
+          },
+          validate: href => {
+            onAddLink && onAddLink(href)
+            return true
           }
         }
-      }),
-
-      PeopleMentions({ onSelection: onAddMention, maxSuggestions, groupIds, suggestionsThemeName }),
+      }
+    }),
 
-      TopicMentions({ onSelection: onAddTopic, maxSuggestions, groupIds, suggestionsThemeName }),
+    PeopleMentions({ onSelection: onAddMention, maxSuggestions, groupIds, suggestionsThemeName }),
 
-      Highlight
-    ],
+    TopicMentions({ onSelection: onAddTopic, maxSuggestions, groupIds, suggestionsThemeName }),
 
-    onBeforeCreate,
+    Highlight
+  ]
 
+  const editor = useEditor({
+    content: contentHTML,
+    extensions,
+    onCreate,
     onUpdate: ({ editor }) => {
       if (!onUpdate) return
-
       onUpdate(editor.getHTML())
     }
-  }, [placeholder, contentHTML]) // TODO: changing the placeholder resets the content of the editor which is probably not what we want
+  })
+
+  // Dynamic setting of initial editor content
```

Comment:
Changes how we set initial content, no longer re-creating the editor instance. This was a necessary part of this change, but is a better practice at this stage with TipTap.